### PR TITLE
Chore: Migrate methods/getRoomInfo to TS

### PR DESCRIPTION
**Pull request by gerzonc**
## Proposed changes

Migrate `methods/getRoomInfo` to TS

## How to test or reproduce

Check room information screen is working properly

## Types of changes

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Improvement (non-breaking change which improves a current function)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Documentation update (if none of the other choices apply)

## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/RocketChat/Rocket.Chat/blob/develop/.github/CONTRIBUTING.md#contributing-to-rocketchat) doc
- [x] I have signed the [CLA](https://cla-assistant.io/RocketChat/Rocket.Chat.ReactNative)
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if applicable)
- [ ] I have added necessary documentation (if applicable)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: app/lib/methods/getRoomInfo.ts [hunk 1]
```diff
@@ -1,7 +1,8 @@
+import { IRoom } from '../../definitions';
 import { getSubscriptionByRoomId } from '../database/services/Subscription';
 import RocketChat from '../rocketchat';
 
-const getRoomInfo = async rid => {
+const getRoomInfo = async (rid: string): Promise<Partial<IRoom> | null> => {
```
Comment (AlexAlexandre) [inline app/lib/methods/getRoomInfo.ts:5]: I don't know if use a `Partial` here is the best approach, take a look at the lint erros.

**Review by AlexAlexandre — CHANGES_REQUESTED** [review]

File: app/lib/methods/getRoomInfo.ts (hunk 1 above)
Comment (gerzonc) [reply app/lib/methods/getRoomInfo.ts:5]: It's a room, just that `getRoomInfo` returns less information than what the `IRoom` interface does, that's why lint complains, we defined `this.room` with an object and some properties from `IRoom`, but on the context we're using it is actually a room with less attributes, that's why I opted for `Partial<IRoom>`.

Comment (AlexAlexandre) [reply app/lib/methods/getRoomInfo.ts:5]: So you can use the `Pick<Type, keys>`. 
Using the keys required at RoomInfo
E.g: Pick<IRoom, 'id' | 'name'>

**Review by AlexAlexandre — APPROVED** [review]
LGTM

**Review by diegolmello — APPROVED** [review]

**Pull request merged**